src/main/scala/org/biodatageeks/datasources/BAM/BAMRelation.scala

```diff
@@ -8,29 +8,28 @@ import org.apache.spark.sql.{Row, SQLContext}
 import org.apache.spark.sql.sources._
 import org.apache.spark.sql.types.{IntegerType, StringType, StructField, StructType}
 import org.seqdoop.hadoop_bam.util.SAMHeaderReader
-import org.seqdoop.hadoop_bam.{BAMInputFormat, FileVirtualSplit, SAMRecordWritable}
+import org.seqdoop.hadoop_bam.{BAMBDGInputFormat, BAMInputFormat, FileVirtualSplit, SAMRecordWritable}
 
+import scala.collection.mutable
 import scala.collection.mutable.ArrayBuffer
 
 
-case class BAMRecord(sampleId: String,
-                     contigName:String,
-                     start:Int,
-                     end:Int,
-                     cigar:String,
-                     mapq:Int,
-                     baseq: String,
-                     reference:String,
-                     flags:Int,
-                     materefind:Int)
+case class BAMBDGRecord(sampleId: String,
```

Comment:
pewnie to ma jakies wytlumaczenie - ale czemu ten refaktoring? struktura klasy jest ta sama wiec to tylko nazwa. BAMRecord bylo spoko ;)